docs/book/inventory.rst

```diff
@@ -52,9 +38,9 @@ Normally, inventory units are created automatically by Sylius and you do not nee
     use Sylius\Component\Inventory\Model\InventoryUnitInterface;
 
     $variant = // Get variant from product.
-    $inventoryUnits = $this->get('sylius.inventory_unit_factory')->create($variant, 6, InventoryUnitInterface::STATE_BACKORDER);
+    $inventoryUnits = $this->get('sylius.inventory_unit_factory')->create($variant, 6, InventoryUnitInterface::STATE_RETURNED);
```

Comment:
We do not support returns out of the box, so the state should be different.

Btw. this state should be completely removed.